desk/sur/vitals.hoon

```diff
@@ -41,4 +41,17 @@
   $:  =ship
       =result
   ==
+::
++$  ping
+  $:  version=@
```

Comment:
This is faux compatibility, I imagine the rest of the type might change if this changes.

But, it has a slightly lower likelihood of breaking whenever `/app/ping` changes, so, I won't oppose it.